### ISSUE TITLE
Seems to not be compatible with aws sso

### ISSUE THREAD

**Issue by HFSVBC** [tsub/s3-edit#22]
Seems to not be compatible with aws sso. When using a profile configured for sso (`aws configure sso`) I am getting the following error:
```
NoCredentialProviders: no valid providers in chain. Deprecated.
        For verbose messaging see aws.Config.CredentialsChainVerboseErrors
```

**Comment by tsub** [comment]
@HFSVBC I have released the commit fixed in PR #23 in [v0.0.16](https://github.com/tsub/s3-edit/releases/tag/v0.0.16).
I do not have an environment where I can test AWS SSO, so it would be helpful if you could check to see if the issue has been resolved.

**Comment by HFSVBC** [comment]
@tsub thank you! it's working perfectly now :D
